release-notes.md

```diff
@@ -2,6 +2,18 @@
 
 [English](./release-notes.en.md)
 
+## [Version 1.1.1](https://github.com/skyway/skyway-android-sdk/releases/tag/v1.1.1)
+
+### Modified
+
+- API認証エラー発生を示すPeerErrorEnum.AUTHENTICATIONをPeerErrorに追加しました
```

Comment:
```suggestion
- API認証エラー発生を示すPeerErrorEnum.AUTHENTICATIONをPeerErrorに追加しました。
```

Reply:
I fixed it.
As it is a small change, I used force-push.